extensions/mapstore/src/test/java/com/hazelcast/mapstore/GenericMapStoreTest.java

```diff
@@ -156,79 +151,6 @@ public void whenMapStoreInitOnNonMaster_thenLoadWaitsForSuccessfulInit() throws
-
-        assertThat(record.getInt32("id")).isZero();
-        assertThat(record.getString("name")).isEqualTo("name-0");
-    }
-
-    @Test
-    public void givenTableMultipleColumns_whenLoad_thenReturnGenericRecord() throws Exception {
-        createTable(mapName, "id INT PRIMARY KEY", "name VARCHAR(100)", "age INT", "address VARCHAR(100)");
-        executeJdbc("INSERT INTO \"" + mapName + "\" VALUES(0, 'name-0', 42, 'Palo Alto, CA 94306')");
-
-        mapStore = createMapStore();
-        GenericRecord record = mapStore.load(0);
-
-        assertThat(record.getInt32("id")).isZero();
-        assertThat(record.getString("name")).isEqualTo("name-0");
-        assertThat(record.getInt32("age")).isEqualTo(42);
-        assertThat(record.getString("address")).isEqualTo("Palo Alto, CA 94306");
-    }
-
-    @Test
-    public void givenTableVarcharPKColumn_whenLoad_thenReturnGenericRecordWithCorrectType() throws Exception {
-        createTable(mapName, "id VARCHAR(100)", "name VARCHAR(100)");
-        executeJdbc("INSERT INTO \"" + mapName + "\" VALUES('0', 'name-0')");
-
-        GenericMapStore<String> mapStore = createMapStore();
-        GenericRecord record = mapStore.load("0");
-
-        assertThat(record.getString("id")).isEqualTo("0");
-        assertThat(record.getString("name")).isEqualTo("name-0");
-        mapStore.destroy();
-    }
-
-    @Test
-    public void givenTable_whenSetColumns_thenGenericRecordHasSetColumns() throws Exception {
-        createTable(mapName, "id INT PRIMARY KEY", "name VARCHAR(100)", "age INT", "address VARCHAR(100)");
-        executeJdbc("INSERT INTO \"" + mapName + "\" VALUES(0, 'name-0', 42, 'Palo Alto, CA 94306')");
-
-        Properties properties = new Properties();
-        properties.setProperty(DATA_LINK_REF_PROPERTY, TEST_DATABASE_REF);
-        properties.setProperty(COLUMNS_PROPERTY, "id,name,age");
-        mapStore = createMapStore(properties, hz);
-
-        GenericRecord record = mapStore.load(0);
-
-        assertThat(record.getInt32("id")).isZero();
-        assertThat(record.getString("name")).isEqualTo("name-0");
-        assertThat(record.getInt32("age")).isEqualTo(42);
-        assertThat(record.getFieldKind("address")).isEqualTo(NOT_AVAILABLE);
-    }
-
-    @Test
-    public void whenSetNonExistingColumn_thenFailToInitialize() throws Exception {
-        createTable(mapName, "id INT PRIMARY KEY", "name VARCHAR(100)");
-        insertItems(mapName, 1);
-
-        Properties properties = new Properties();
-        properties.setProperty(DATA_LINK_REF_PROPERTY, TEST_DATABASE_REF);
-
-        properties.setProperty("columns", "name,age");
-        mapStore = createMapStore(properties, hz);
-
-        assertThatThrownBy(() -> mapStore.load(0))
-                .isInstanceOf(HazelcastException.class)
-                .hasStackTraceContaining("Column 'age' not found");
-    }
-
     @Test
     public void whenSetNonExistingColumnOnSecondMapStore_thenFailToInitialize() throws Exception {
```

Comment:
Why isn't this test in the `GenericMapLoaderTest`? The GenericMapLoader should validate the columns as well.